contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -107,7 +107,9 @@
       },
       canSave() {
         if (this.hasMixedCategories) {
-          return Object.values(this.selectedValues).some(value => value.length > 0);
+          return Object.values(this.selectedValues).some(
+            value => value.length === this.nodes.length
```

Comment:
Hey! The thing is that "partial selections" just only serve to determine if we have mixedCategories, but the user doesnt see nor interact with any of these partial selections. For them they dont exists, so thats why we cant depend on them, and rather depend on what they are seeing which is selected options, and we have selected options when we have that `value.length === this.nodes.length`.

If we keep the condition of `value.length > 0` then we will be taking these mixed options as valid values. So if I have for example 2 nodes:

Node 1:
* Category A selected

Node 2:
* Category B selected

Then when we open the modal, we will see the mixed options comment, but we wont see any of the options selected, because we dont have any option selected across all nodes, and we wont see any indeterminate state as they dont exist anymore, the only the user will se will be all checkboxes unchecked. But the user will still see the save button enabled, because `canSave = true` because all category A and B have `value.length > 0`, and this is an error, as we are not meeting the 4th acceptance criteria mentioned in #4651:
> the "save" button only be active if one or more boxes is checked, but no validation is required (and no error message)

So, no, we cant have a more flexible condition as this condition does not met the requirements.

Reply:
Thank you for explaining the reasoning behind the stricter condition.